app.js

```diff
@@ -0,0 +1,20 @@
+// We create  own server named app
+const express = require('express');
+
+// Express server handling requests and responses
+const app = express();
+
+// Make everything inside of public/ available
+app.use(express.static('public'));
+
+//home page Route:
+app.get('/', (request, response, next) => response.sendFile(__dirname + '/views/home-page.html'));
+
```

Comment:
Esta forma está bien, pero mejor que utilicéis esta otra, ya que con render puedes pasar parámetros :

app.get('/', (req, res, next) => res.render('home'))

También res, req, next, es igual de legible y lo verás siempre así.